Link cstool with capstone-shared instead of -static
--- capstone-4.0.1/CMakeLists.txt.orig	2019-01-10 14:45:56.000000000 +0100
+++ capstone-4.0.1/CMakeLists.txt	2020-05-23 20:25:12.603569843 +0200
@@ -491,7 +491,6 @@
 if (CAPSTONE_BUILD_STATIC)
     add_library(capstone-static STATIC ${ALL_SOURCES} ${ALL_HEADERS})
     set_property(TARGET capstone-static PROPERTY OUTPUT_NAME capstone)
-    set(default-target capstone-static)
 endif ()
 
 # Force static runtime libraries
